sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/runtime/ManifestFileCommitProtocol.scala

```diff
@@ -114,13 +114,13 @@ class ManifestFileCommitProtocol(jobId: String, path: String)
   }
 
   override def newTaskTempFile(
-      taskContext: TaskAttemptContext, dir: Option[String], ext: String): String = {
+      taskContext: TaskAttemptContext, dir: Option[String], spec: FileNameSpec): String = {
```

Comment:
Hi, @yaooqinn . Can we spin off this change independently? We had better focus on `ManifestFileCommitProtocol` change.

Reply:
This is a necessary step， we need to override the new method

Reply:
Yes, @yaooqinn . However, we don't need to remove inner references of deprecated APIs when we switch API in this layer. That's the reason why I requested to spin off. We can throw exception in this PR but not in the spin-off PR.